userspace/falco/falco_metrics.cpp

```diff
@@ -86,26 +86,59 @@ std::string falco_metrics::to_text(const falco::app::state& state)
 		{
 			prometheus_text += prometheus_metrics_converter.convert_metric_to_text_prometheus("evt_source", "falcosecurity", "falco", {{"evt_source", source}});
 		}
-		std::vector<metrics_v2> static_metrics;
-		static_metrics.push_back(libs_metrics_collector.new_metric("start_ts",
+		std::vector<metrics_v2> falco_metrics;
```

Comment:
It would be nice to split this method into multiple logical pieces, like:
* scap_engines_to_text()
* falco_rules_to_text()
* agent_info_to_text()
*
...and so on.
Do you think it is feasible?

Reply:
Of course. Bog question is now or wait until a more major cleanup follows post Falco 0.38.0, cc @sgaist?

Reply:
Much better, thanks!